fdm-app/app/routes/farm.$b_id_farm.settings.properties.tsx

```diff
@@ -0,0 +1,216 @@
+Wageningen"
+                                                    className="resize-none"
+                                                    autoComplete="address"
+                                                    rows={3}
+                                                    maxLength={300}
+                                                    {...field}
+                                                />
+                                            </FormControl>
+                                            <FormDescription />
+                                            <FormMessage />
+                                        </FormItem>
+                                    )}
+                                />
+                            </div>
+                            <div className="flex flex-col space-y-1.5 col-span-1">
+                                <FormField
+                                    control={form.control}
+                                    name="b_postalcode_farm"
+                                    render={({ field }) => (
+                                        <FormItem>
+                                            <FormLabel>Postcode</FormLabel>
+                                            <FormControl>
+                                                <Input placeholder="bv. 1234 AB" {...field} autoComplete="postal-code" maxLength={300} />
+                                            </FormControl>
+                                            <FormDescription />
+                                            <FormMessage />
+                                        </FormItem>
+                                    )}
+                                />
+                            </div>
+                        </div >
+                    </fieldset>
+                    <br />
+                    <div className="ml-auto">
+                        <Button
+                            type="submit"
+                            disabled={form.formState.isSubmitting}
+                            className="m-auto"
+                        >
+                            {form.formState.isSubmitting && <LoadingSpinner />}
+                            Bijwerken
+                        </Button>
+                    </div>
+
+                </Form>
+            </RemixFormProvider>
+        </div>
+    )
+}
+
+export async function action({ request, params }: ActionFunctionArgs) {
+    const b_id_farm = params.b_id_farm;
+
+    if (!b_id_farm) {
+        return dataWithError(null, "Missing farm ID.");
+    }
+
+    try {
+        const formValues = await extractFormValuesFromRequest(request, FormSchema);
+
+        await updateFarm(
+            fdm,
+            b_id_farm,
+            formValues.b_name_farm,
+            formValues.b_businessid_farm,
+            formValues.b_address_farm,
+            formValues.b_postalcode_farm
+        );
```

Comment:
email-adres user is independent of farm?

Reply:
Yes, the user has an email not the farm. In the future the general farm email adress could be of the users with the role "owner" for that specific farm